.github/scripts/pr_body_check.py

```
@@ -0,0 +1,64 @@
+import os
+import sys
+import re
+
+def warn(msg):
```

Ruff (0.12.2)

Missing return type annotation for public function `warn`

Add return type annotation: `None`

(ANN201)